qml.qrc

```diff
@@ -33,6 +33,8 @@
         <file>entrypoint/GenericEntry.qml</file>
         <file>backend/sccs/PluginSettings.qml</file>
         <file>ui/sccs/PluginSettings.qml</file>
+        <file>ui/pages/ProjectParametersPage.qml</file>
+        <file>ui/pages/ProjectDeployPage.qml</file>
```

Comment:
All UIs components relative to continuous deployment  should be renamed to fit <something>ContinuousDeployment<something else> like in Sccs folder for backend and ui.
This is to keep the project more readable.